README.md

```diff
@@ -641,264 +576,243 @@ Convert an array into DoubleLinker instances, return the head and tail DoubleLin
 
 <a name="LinkedTreeList"></a>
 
-## LinkedTreeList ⇐ [<code>DoublyLinkedList</code>](#DoublyLinkedList)
-LinkedTreeList represents a collection stored with a root and spreading in branching (tree) formation..
+## LinkedTreeList
+LinkedTreeList represents a collection stored with a root and spreading in branching (tree) formation.
 
 **Kind**: global class  
-**Extends**: [<code>DoublyLinkedList</code>](#DoublyLinkedList)  
 
-* [LinkedTreeList](#LinkedTreeList) ⇐ [<code>DoublyLinkedList</code>](#DoublyLinkedList)
-    * [new LinkedTreeList([listClass])](#new_LinkedTreeList_new)
+* [LinkedTreeList](#LinkedTreeList)
+    * [new LinkedTreeList()](#new_LinkedTreeList_new)
     * _instance_
+        * [.list](#LinkedTreeList+list) ⇒ [<code>TreeLinker</code>](#TreeLinker)
+        * [.first](#LinkedTreeList+first) ⇒ [<code>TreeLinker</code>](#TreeLinker)
+        * [.last](#LinkedTreeList+last) ⇒ [<code>TreeLinker</code>](#TreeLinker)
+        * [.length](#LinkedTreeList+length) ⇒ <code>number</code>
+        * [.children](#LinkedTreeList+children)
+        * [.children](#LinkedTreeList+children)
```

Comment:
Investigate the cause of this duplication, It seems that get and set for children and parent both cause these duplicates.